toc/working-groups/app-runtime-platform.md

```diff
@@ -26,14 +26,19 @@ Provides operational components for the CF App Runtime, including those for appl
 
 ## Non-Goals
 
-
+- Be responsible for these functional domain areas outside of the CF App Runtime system.
 
 
 ## Proposed Membership
 
 - Technical Lead(s): @ameowlia, others TBD
 - Execution Lead(s): TBD
-- Approvers: TBD
+- Approvers by area:
+  - Diego: @acrmp, @aminjam, @Benjamintf1, @jrussett, @pivotalgeorge, @reneighbor, @selzoc
+  - Eirini: @danail-branekov, @gcapizzi, @georgethebeatle, @kieron-dev, @mnitchev
+  - Garden Containers: @aemengo, @danail-branekov, @gcapizzi, @georgethebeatle, @kieron-dev, @mnitchev
+  - Logging and Metrics: @acrmp, @aminjam, @Benjamintf1, @jrussett, @pivotalgeorge, @reneighbor, @selzoc
```

Comment:
Can we add @geofffranks to Diego and Logging+Metics,

Reply:
Done!